### ISSUE TITLE
[My Daily Log] Add ability to edit log line

### ISSUE THREAD

**Issue by boaz-wiesner** [raycast/extensions#5042]
### Extension

https://www.raycast.com/frugoman/my-daily-log

### Description

I'm an avid user of this extension, it's super simple and perfect for my daily use, so thanks so much for making it!
One thing that happens frequently is that I mistype something in my log and don't notice until afterwards. It's not the end of the world, since the logs are for myself, so I generally understand what I meant, but if there was a way to edit or delete loglines from within raycast that would be amazing.

Thanks again for all your work!

**Comment by frugoman** [comment]
Hey @boaz-wiesner Thanks for this!!

I've revamped the extension, rewriting most of it following clean architecture and some other behind the scenes changes. While at this, I've added support for editing the logs!

The refactor was done mainly because of this, as it wasn't even possible before because of the initial state of the project's code 😬, so thanks for giving me a reason to finally get back to it and fix all this!!

https://github.com/raycast/extensions/pull/5389

Hopefully will be merged soon!
